CONTRIBUTING.md

```diff
@@ -25,10 +28,29 @@
 
 # Contributing to Athens
 
-Whether you are a designer, developer, or have other superpowers, please see our [v1 Project Board](https://github.com/athensresearch/athens/projects/2) to see what we're working on.
+## Developers
 
-- The best place to reach us is our [Discord](https://discord.gg/GCJaV3V)! 👾
+- Join our [Discord](https://discord.gg/GCJaV3V) 👾 and check out the `#engineering` and `#engineers` channels.
```

Comment:
Do we want to add a little note on how the two channels are different from one another?